src/main/java/mate/controller/GetAllCarsController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet("/cars/all")
+public class GetAllCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Car> cars = carService.getAll();
+        req.setAttribute("cars", cars);
+        req.getRequestDispatcher("/WEB-INF/views/car/allCars.jsp").forward(req, resp);
```

Comment:
Let's simplify `jsp` pages names a bit
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/car/cars.jsp").forward(req, resp);
```
Please fix all similar cases.